src/test/ui/issue-13483.stderr

```diff
@@ -0,0 +1,18 @@
+error: missing conditional
+  --> $DIR/issue-13483.rs:13:15
+   |
+13 |       } else if { //ERROR: MISSING CONDITIONAL
```

Comment:
I agree with @Mark-Simulacrum that the span here is not ideal -- also, the term "conditional" doesn't quite feel right. I feel like I would prefer:

```rust
error: missing condition for `if` statemement
  --> $DIR/issue-13483.rs:20:15
   |
20 |       } else if {
   |               ^ expected if condition here
21 | |     };
```